### PR TITLE
Login was being used as a verb in some places.

### PR DESCRIPTION
**Pull request by wcjr**
see Trello board.

### CODE REVIEW

File: dictionaries/login.definition.json [hunk 1]
```diff
@@ -29,10 +29,10 @@
 		"fr": "Quelque chose ne va pas avec ce nom d'utilisateur ou ce mot de passe, veuillez vérifier et essayer à nouveau.",
 		"ko": "해당 사용자 이름 또는 비밀번호가 잘못되었습니다. 다시 확인하고 다시 시도하십시오."
 	},
-	"button_login": {
-		"en": "Login",
+	"button_log_in": {
```
Comment (Baggerone) [inline dictionaries/login.definition.json]: You don't really need to change the key for the translation, do you?

https://developers.facebook.com/docs/facebook-login/web/login-button  :-)

File: themes/material/core/loginuserpass.php [hunk 2]
```diff
@@ -19,9 +19,9 @@ function submitForm() {
         }
 
         function onRecaptchaLoad() {
-            var loginButton = document.querySelector('button');
+            var logInButton = document.querySelector('button');
 
-            grecaptcha.render(loginButton, {
+            grecaptcha.render(logInButtonn, {
```
Comment (Baggerone) [inline themes/material/core/loginuserpass.php]: typo?  "logInButtonn" has two n's.

In any case, I don't see the point in making these two changes.

File: themes/material/core/loginuserpass.php [hunk 3]
```diff
@@ -123,7 +123,7 @@ class="mdl-button mdl-button--colored mdl-typography--caption">
                     <span flex></span>
 
                     <button class="mdl-button mdl-button--colored mdl-button--raised">
-                        <?= $this->t('{material:login:button_login}') ?>
+                        <?= $this->t('{material:login:button_log_in}') ?>
```
Comment (Baggerone) [inline themes/material/core/loginuserpass.php]: I suggest you leave it as "button_login"

File: dictionaries/login.definition.json (hunk 1 above)
Comment (wcjr) [inline dictionaries/login.definition.json]: I definitely don't _need_ to.

File: themes/material/core/loginuserpass.php (hunk 2 above)
Comment (wcjr) [inline themes/material/core/loginuserpass.php]: Thanks, I'll take care of that.

File: themes/material/core/loginuserpass.php (hunk 3 above)
Comment (wcjr) [inline themes/material/core/loginuserpass.php]: 👍

File: dictionaries/login.definition.json [hunk 4]
```diff
@@ -29,10 +29,10 @@
...
+		"en": "Log in",
 		"es": "Iniciar sesión",
-		"fr": "Connexion",
```
Comment (Baggerone) [inline dictionaries/login.definition.json]: The French should probably be "Se connecter"

**Comment by wcjr** [comment]
going to go ahead and abandon this for now in hopes @forevermatt will either open an issue for later or be ok with it the way it is.

**Pull request closed without merging**